### ISSUE TITLE
Can't build nyxt

### ISSUE THREAD

**Issue by franburstall** [atlas-engineer/nyxt#1529]
**Describe the bug**

Since https://github.com/atlas-engineer/nyxt/commit/700cf5449ea57ac46dd8c9633a6cc37333b4e8ad, I can't build nyxt from source.  I get the following error:
````
; caught ERROR:
;   READ error during COMPILE-FILE:
;   
;     Symbol "WEBKIT-SCRIPT-DIALOG-REF" not found in the CL-WEBKIT2 package.
;   
;       Line: 829, Column: 39, File-Position: 37033
;   
````
Complete transcript of build at the end.

**Information**
- OS name+version: Debian 10
- Graphics card and driver:
- Desktop environment / Window manager name+version: EXWM
- How you installed Nyxt (Guix pack, package manager, build from source): source

**Output when started from a shell**
demoulin:~/c/nyxt% make all
sbcl --no-userinit --non-interactive --eval '(require "asdf")' --eval '(when (string= "true" "true") (setf asdf:*default-source-registries* nil) (asdf:clear-configuration) (asdf:load-asd "/home/fran/common-lisp/nyxt//nyxt-quicklisp.asd") (asdf:load-system :nyxt-quicklisp))' --eval '(asdf:load-asd "/home/fran/common-lisp/nyxt//nyxt.asd")' --eval '(when (find-package :ql) (funcall (read-from-string "ql:quickload") :cffi))' --eval '(when (and (find-package :cffi) (uiop:getenv "GUIX_ENVIRONMENT")) (pushnew (pathname (format nil "~a/lib/" (uiop:getenv "GUIX_ENVIRONMENT"))) (symbol-value (read-from-string "cffi:*foreign-library-directories*" )) :test (quote equal)))' --eval '(ql:quickload :nyxt/gi-gtk-application)' \
        --eval '(asdf:make :nyxt/gi-gtk-application)' \
        --eval '(uiop:quit)' || (printf "\n%s\n%s\n" "Compilation failed, see the above stacktrace." && exit 1)
This is SBCL 2.1.1, an implementation of ANSI Common Lisp.
More information about SBCL is available at <http://www.sbcl.org/>.

SBCL is free software, provided as is, with absolutely no warranty.
It is mostly in the public domain; some portions are provided under
BSD-style licenses.  See the CREDITS and COPYING files in the
distribution for more information.
Submodule path '_build/quicklisp-client': checked out 'b525ae5a4db984bf96a33b9c7379b3b13a4edd98'
Submodule path '_build/submodules/cl-cffi-gtk': checked out 'e9a46df65995d9a16e6c8dbdc1e09b775eb4a966'
Submodule path '_build/submodules/cl-webkit': checked out '29aeb74d854bd2ac52959ef3d793389d238636c5'
Submodule path '_build/submodules/cluffer': checked out '4aad29c276a58a593064e79972ee4d77cae0af4a'
; CL_SOURCE_REGISTRY: "/home/fran/common-lisp/nyxt/_build/submodules//:/home/fran/common-lisp/nyxt/:"
You already have the latest version of "quicklisp": 2021-05-31.
To load "cffi":
  Load 1 ASDF system:
    cffi
; Loading "cffi"
.
To load "nyxt/gi-gtk-application":
  Load 1 ASDF system:
    nyxt/gi-gtk-application
; Loading "nyxt/gi-gtk-application"
..................................................
[package class-star]..............................
..................................................
..................................................
..................................................
..................................................
..................................................
......................................; 
; caught ERROR:
;   READ error during COMPILE-FILE:
;   
;     Symbol "WEBKIT-SCRIPT-DIALOG-REF" not found in the CL-WEBKIT2 package.
;   
;       Line: 829, Column: 39, File-Position: 37033
;   
;       Stream: #<SB-INT:FORM-TRACKING-STREAM for "file /home/fran/common-lisp/nyxt/source/renderer-gtk.lisp" {100B8B1403}>
Unhandled UIOP/LISP-BUILD:COMPILE-FILE-ERROR in thread #<SB-THREAD:THREAD "main thread" RUNNING
                                                          {1001550283}>:
  COMPILE-FILE-ERROR while compiling #<CL-SOURCE-FILE "nyxt/gtk" "renderer-gtk">

Backtrace for: #<SB-THREAD:THREAD "main thread" RUNNING {1001550283}>
0: (SB-DEBUG::DEBUGGER-DISABLED-HOOK #<UIOP/LISP-BUILD:COMPILE-FILE-ERROR {1007E86C33}> #<unused argument> :QUIT T)
1: (SB-DEBUG::RUN-HOOK *INVOKE-DEBUGGER-HOOK* #<UIOP/LISP-BUILD:COMPILE-FILE-ERROR {1007E86C33}>)
2: (INVOKE-DEBUGGER #<UIOP/LISP-BUILD:COMPILE-FILE-ERROR {1007E86C33}>)
3: (ERROR UIOP/LISP-BUILD:COMPILE-FILE-ERROR :CONTEXT-FORMAT "~/asdf-action::format-action/" :CONTEXT-ARGUMENTS ((#<ASDF/LISP-ACTION:COMPILE-OP > . #<ASDF/LISP-ACTION:CL-SOURCE-FILE "nyxt/gtk" "renderer-gtk">)))
4: (UIOP/LISP-BUILD:CHECK-LISP-COMPILE-RESULTS NIL T T "~/asdf-action::format-action/" ((#<ASDF/LISP-ACTION:COMPILE-OP > . #<ASDF/LISP-ACTION:CL-SOURCE-FILE "nyxt/gtk" "renderer-gtk">)))
5: ((SB-PCL::EMF ASDF/ACTION:PERFORM) #<unused argument> #<unused argument> #<ASDF/LISP-ACTION:COMPILE-OP > #<ASDF/LISP-ACTION:CL-SOURCE-FILE "nyxt/gtk" "renderer-gtk">)
6: ((LAMBDA NIL :IN ASDF/ACTION:CALL-WHILE-VISITING-ACTION))
7: ((:METHOD ASDF/ACTION:PERFORM-WITH-RESTARTS :AROUND (T T)) #<ASDF/LISP-ACTION:COMPILE-OP > #<ASDF/LISP-ACTION:CL-SOURCE-FILE "nyxt/gtk" "renderer-gtk">) [fast-method]
8: ((:METHOD ASDF/PLAN:PERFORM-PLAN (T)) #<ASDF/PLAN:SEQUENTIAL-PLAN {1003FCB7E3}>) [fast-method]
9: ((FLET SB-C::WITH-IT :IN SB-C::%WITH-COMPILATION-UNIT))
10: ((:METHOD ASDF/PLAN:PERFORM-PLAN :AROUND (T)) #<ASDF/PLAN:SEQUENTIAL-PLAN {1003FCB7E3}>) [fast-method]
11: ((:METHOD ASDF/OPERATE:OPERATE (ASDF/OPERATION:OPERATION ASDF/COMPONENT:COMPONENT)) #<ASDF/LISP-ACTION:LOAD-OP > #<ASDF/SYSTEM:SYSTEM "nyxt/gi-gtk-application"> :PLAN-CLASS NIL :PLAN-OPTIONS NIL) [fast-method]
12: ((SB-PCL::EMF ASDF/OPERATE:OPERATE) #<unused argument> #<unused argument> #<ASDF/LISP-ACTION:LOAD-OP > #<ASDF/SYSTEM:SYSTEM "nyxt/gi-gtk-application"> :VERBOSE NIL)
13: ((LAMBDA NIL :IN ASDF/OPERATE:OPERATE))
14: ((:METHOD ASDF/OPERATE:OPERATE :AROUND (T T)) #<ASDF/LISP-ACTION:LOAD-OP > #<ASDF/SYSTEM:SYSTEM "nyxt/gi-gtk-application"> :VERBOSE NIL) [fast-method]
15: ((SB-PCL::EMF ASDF/OPERATE:OPERATE) #<unused argument> #<unused argument> ASDF/LISP-ACTION:LOAD-OP "nyxt/gi-gtk-application" :VERBOSE NIL)
16: ((LAMBDA NIL :IN ASDF/OPERATE:OPERATE))
17: ((:METHOD ASDF/OPERATE:OPERATE :AROUND (T T)) ASDF/LISP-ACTION:LOAD-OP "nyxt/gi-gtk-application" :VERBOSE NIL) [fast-method]
18: (ASDF/SESSION:CALL-WITH-ASDF-SESSION #<FUNCTION (LAMBDA NIL :IN ASDF/OPERATE:OPERATE) {1003FB845B}> :OVERRIDE T :KEY NIL :OVERRIDE-CACHE T :OVERRIDE-FORCING NIL)
19: ((LAMBDA NIL :IN ASDF/OPERATE:OPERATE))
20: (ASDF/SESSION:CALL-WITH-ASDF-SESSION #<FUNCTION (LAMBDA NIL :IN ASDF/OPERATE:OPERATE) {1003FABEFB}> :OVERRIDE NIL :KEY NIL :OVERRIDE-CACHE NIL :OVERRIDE-FORCING NIL)
21: ((:METHOD ASDF/OPERATE:OPERATE :AROUND (T T)) ASDF/LISP-ACTION:LOAD-OP "nyxt/gi-gtk-application" :VERBOSE NIL) [fast-method]
22: (ASDF/OPERATE:LOAD-SYSTEM "nyxt/gi-gtk-application" :VERBOSE NIL)
23: (QUICKLISP-CLIENT::CALL-WITH-MACROEXPAND-PROGRESS #<FUNCTION (LAMBDA NIL :IN QUICKLISP-CLIENT::APPLY-LOAD-STRATEGY) {1003FABE4B}>)
24: (QUICKLISP-CLIENT::AUTOLOAD-SYSTEM-AND-DEPENDENCIES "nyxt/gi-gtk-application" :PROMPT NIL)
25: ((:METHOD QL-IMPL-UTIL::%CALL-WITH-QUIET-COMPILATION (T T)) #<unused argument> #<FUNCTION (FLET QUICKLISP-CLIENT::QL :IN QUICKLISP-CLIENT:QUICKLOAD) {1003F8D3BB}>) [fast-method]
26: ((:METHOD QL-IMPL-UTIL::%CALL-WITH-QUIET-COMPILATION :AROUND (QL-IMPL:SBCL T)) #<QL-IMPL:SBCL {100508F263}> #<FUNCTION (FLET QUICKLISP-CLIENT::QL :IN QUICKLISP-CLIENT:QUICKLOAD) {1003F8D3BB}>) [fast-method]
27: ((:METHOD QUICKLISP-CLIENT:QUICKLOAD (T)) :NYXT/GI-GTK-APPLICATION :PROMPT NIL :SILENT NIL :VERBOSE NIL) [fast-method]
28: (QL-DIST::CALL-WITH-CONSISTENT-DISTS #<FUNCTION (LAMBDA NIL :IN QUICKLISP-CLIENT:QUICKLOAD) {1003F864DB}>)
29: (SB-INT:SIMPLE-EVAL-IN-LEXENV (QUICKLISP-CLIENT:QUICKLOAD :NYXT/GI-GTK-APPLICATION) #<NULL-LEXENV>)
30: (EVAL (QUICKLISP-CLIENT:QUICKLOAD :NYXT/GI-GTK-APPLICATION))
31: (SB-IMPL::PROCESS-EVAL/LOAD-OPTIONS ((:EVAL . "(require \"asdf\")") (:EVAL . #<(SIMPLE-ARRAY CHARACTER (202)) (when (string= "true" "true") (setf asdf:*default-source-registries* nil) (asdf:clear-configuration) (asdf:load-asd "/home/fran/common-lisp/nyxt//nyxt-quicklisp.asd") (asdf:load-system :nyxt-quicklisp... {10047A02AF}>) (:EVAL . "(asdf:load-asd \"/home/fran/common-lisp/nyxt//nyxt.asd\")") (:EVAL . "(when (find-package :ql) (funcall (read-from-string \"ql:quickload\") :cffi))") (:EVAL . #<(SIMPLE-ARRAY CHARACTER (235)) (when (and (find-package :cffi) (uiop:getenv "GUIX_ENVIRONMENT")) (pushnew (pathname (format nil "~a/lib/" (uiop:getenv "GUIX_ENVIRONMENT"))) (symbol-value (read-from-string "cffi:*foreign-library-dir... {10047A081F}>) (:EVAL . "(ql:quickload :nyxt/gi-gtk-application)") (:EVAL . "(asdf:make :nyxt/gi-gtk-application)") (:EVAL . "(uiop:quit)") (:QUIT)))
32: (SB-IMPL::TOPLEVEL-INIT)
33: ((FLET SB-UNIX::BODY :IN SB-IMPL::START-LISP))
34: ((FLET "WITHOUT-INTERRUPTS-BODY-1" :IN SB-IMPL::START-LISP))
35: (SB-IMPL::START-LISP)

unhandled condition in --disable-debugger mode, quitting
; 
; compilation unit aborted
;   caught 2 fatal ERROR conditions
;   caught 1 ERROR condition

Compilation failed, see the above stacktrace.

make: *** [Makefile:41: nyxt] Error 1

**Comment by Ambrevar** [comment]
Thanks for reporting, should be fixed with 4ca0a08dee003ea8fd87de54362fe23eb5162269.

**Comment by franburstall** [comment]
Yup--that does the job.
